### PR TITLE
add userId to bids sent to Causality

### PR DESCRIPTION
**Pull request by acjay**
Also update snapshot images

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5271,16 +5267,12 @@
...
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Unica77LL-Italic.otf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Unica77LL-Medium.otf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Unica77LL-MediumItalic.otf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Unica77LL-Regular.otf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EBGaramond08-Italic.ttf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EBGaramond08-Regular.ttf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EBGaramond12-Italic.ttf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EBGaramond12-Regular.ttf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/texgyreadventor-bold.ttf",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/texgyreadventor-regular.ttf",
```
Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: It looks like these changes may still be left over from when you were using the OSS fonts. Can you revert this?

Comment (acjay) [reply Artsy.xcodeproj/project.pbxproj]: Ah, yep.

File: Artsy/App/ARSwitchBoard.m [hunk 2]
```diff
@@ -41,7 +41,7 @@
 NSString *const AREscapeSandboxQueryString = @"eigen_escape_sandbox";
 
 /// To be kept in lock-step with the corresponding echo value, and updated when there is a breaking causality change.
-NSInteger const ARLiveAuctionsCurrentWebSocketVersionCompatibility = 3;
+NSInteger const ARLiveAuctionsCurrentWebSocketVersionCompatibility = 4;
```
Comment (acjay) [inline Artsy/App/ARSwitchBoard.m]: @ashfurrow Let me know if this is right. Thanks!

Comment (ashfurrow) [reply Artsy/App/ARSwitchBoard.m]: Yup! Gotcha, so `userId` will be required by Causality going forward? Once that requirement is in place and we have released this code to the store, we can update the [echo](https://github.com/artsy/echo) config `LiveAuctionsCurrentWebSocketVersion` value to `4` to tell older versions of Eigen to fallback to Prediction's mobile web UI.

Comment (acjay) [reply Artsy/App/ARSwitchBoard.m]: Yeah. It seems like we'll want to make the Echo change right before dropping optionality in Causality.

Comment (ashfurrow) [reply Artsy/App/ARSwitchBoard.m]: Yup, perfect.

**Comment by alloy** [comment]
Aight, looks good then 👍

**Comment by orta** [comment]
Nice work

**Pull request merged**